README.md

```diff
@@ -163,8 +176,21 @@ For example
 
 Note: Each `varint-length` contains the size of the rest of the line, including the newline bytes
 
-# Implementations
+## Implementations
+
+- [js-multistream-select](https://github.com/multiformats/js-multistream-select) - JavaScript Implementation
+- [mss-nc](https://github.com/whyrusleeping/mss-nc) - multistream netcat written in Go
+
+## Maintainers
+
+Captain: [@jbenet](https://github.com/jbenet).
+
+## Contribute
+
+Contributions welcome. Please check out [the issues](https://github.com/multiformats/multistream-select/issues).
+
+Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+## License
 
-- [js-multistream-select](https://github.com/diasdavid/js-multistream-select) - JavaScript Implementation
-- [go-multistream-select](https://github.com/whyrusleeping/go-multistream-select) - Go Implementation
```

Comment:
Oh right, it is just go-multistream
https://github.com/multiformats/go-multistream